device-types/APC/SMT1000RMI2UC.yaml

```diff
@@ -0,0 +1,28 @@
+---
+manufacturer: APC
+model: Smart-UPS 1000VA, Rack Mount, LCD 230V with SmartConnect Port
+slug: smart-ups-1000va-rack-mount-lcd-230-with-smartconnect-port
+part_number: SMT1000RMI2UC
+u_height: 2
+is_full_depth: true
```

Comment:
Please add airflow if you can

Reply:
added weight/weight_unit. Checked if float (22.5) is accepted.
As requested, added prefix apc- and airflow. Technically there is not much of airflow, the UPS just has a fan on its power supply on the rear.